sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+    "- West Europe\n",
+    "- Central US EUAP"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aacs_location = \"east us\" # please choose the nearest location to your workspace\n",
+    "print(f\"will create aacs in {aacs_location}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.mgmt.cognitiveservices import CognitiveServicesManagementClient\n",
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "from azure.mgmt.cognitiveservices.models import Account, Sku, AccountProperties\n",
+    "import time\n",
+    "\n",
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    # Check if given credential can get token successfully.\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()\n",
+    "\n",
+    "client = CognitiveServicesManagementClient(credential, subscription_id)\n",
+    "# create a new Cognitive Services Account\n",
+    "# name of the AACS to create\n",
+    "name_of_aacs = f\"{endpoint_name}-aacs\"\n",
+    "kind = \"ContentSafety\"\n",
+    "sku_name = \"S0\"\n",
+    "parameters = Account(sku=Sku(name=sku_name), kind=kind, location=aacs_location, properties= AccountProperties(custom_sub_domain_name=name_of_aacs, public_network_access=\"Enabled\"))\n",
+    "# How many seconds to wait between checking the status of an async operation.\n",
+    "wait_time = 10\n",
+    "\n",
+    "poller = client.accounts.begin_create(resource_group_name, name_of_aacs, parameters)\n",
+    "while (False == poller.done()) :\n",
+    "    print (\"Waiting {wait_time} seconds for operation to finish.\".format (wait_time = wait_time))\n",
+    "    time.sleep (wait_time)\n",
+    "    # This will raise an exception if the server responded with an error.\n",
+    "    result = poller.result()\n",
+    "\n",
+    "\n",
+    "print(\"Resource created.\")\n",
+    "\n",
+    "aacs=client.accounts.get(resource_group_name, name_of_aacs)\n",
+    "aacs_endpoint = aacs.properties.endpoint\n",
+    "aacs_resource_id = aacs.id\n",
+    "print(aacs_endpoint)\n",
+    "print(aacs_resource_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Create LLaMa Oneline endpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 5.1 Decide on SKU and instance count for the LLama Oneline endpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "compute_sku_for_llama=\"Standard_DS5_v2\" # the sku of the compute instance for LLaMa endpoint\n",
```

Comment:
While I think we should only use 1 endpoint, either way we should provide some SKU recommendation and instance count recommendations here in the notebook for the user, and guide them to pick a suitable one.

The recommendations should be there on the model tags already.

Reply:
Added recommendation